UPGRADE-8.0.md

```diff
@@ -330,6 +330,25 @@ Security
  * Remove `AbstractListener::__invoke`
  * Remove `LazyFirewallContext::__invoke()`
 
+SecurityBundle
+--------------
+
+ * Remove the deprecated `algorithm` and `key` options from the OIDC token handler configuration, use `algorithms` and `keyset` instead
+
+   ```diff
+   # config/packages/security.yaml
+   security:
+       firewalls:
+           main:
+               access_token:
+                   token_handler:
+                       oidc:
+   -                        algorithm: 'RS256'
+   -                        key: 'https://example.com/.well-known/jwks.json'
+   +                        algorithms: ['RS256']
+   +                        keyset: 'https://example.com/.well-known/jwks.json'
+   ```
```

Comment:
in a diff, non modified lines must start with an extra space (instead if a + / - for changed lines):
```suggestion
    # config/packages/security.yaml
    security:
        firewalls:
            main:
                access_token:
                    token_handler:
                        oidc:
   -                        algorithm: 'RS256'
   -                        key: 'https://example.com/.well-known/jwks.json'
   +                        algorithms: ['RS256']
   +                        keyset: 'https://example.com/.well-known/jwks.json'
   ```
```